src/main/java/org/spongepowered/api/event/SpongeEventFactory.java

```diff
@@ -951,11 +951,8 @@ public static PlayerBreakBlockEvent createPlayerBreakBlock(Game game, Cause caus
         values.put("cause", Optional.fromNullable(cause));
         values.put("block", block);
         values.put("entity", player);
-        values.put("replacementBlock", replacementBlock);
-        values.put("player", player);
         values.put("user", player);
-        values.put("human", player);
-        values.put("living", player);
+        values.put("replacementBlock", replacementBlock);
```

Comment:
Thats an issue of my needing to add and remove the value for user. Seemingly I readded them at the wrong spot. Arg